src/main/kotlin/com/sourcegraph/cody/vscode/InlineAutocompleteItem.kt

```diff
@@ -1,8 +1,8 @@
 package com.sourcegraph.cody.vscode
 
 class InlineAutocompleteItem(
-    val insertText: String,
+    var insertText: String,
     val filterText: String,
-    val range: Range,
+    var range: Range,
     val command: Command
```

Comment:
I do not like this var-val mix here 😕 maybe all these fields should be var? or it could be a data class? wdyt?